src/components/event.tsx

```diff
@@ -11,30 +11,30 @@ export default function Event({ event }: { event: AppBooking }) {
           </div>
           <div className='eventDescription'>{event.summary}</div>
         </div>
-        <EventTimeComponent start={new Date(event.startDateTime)} end={new Date(event.endDateTime)} />
+        <EventTimeComponent
+          start={new Date(event.startDateTime)}
+          end={new Date(event.endDateTime)}
+        />
       </div>
     </div>
   );
 }
 
 const isBookingAllDay = (start: Date, end: Date): boolean => {
   const dayInMilliseconds = 1000 * 60 * 60 * 24;
-  return (end.valueOf() - start.valueOf()) == dayInMilliseconds;
+  return end.valueOf() - start.valueOf() == dayInMilliseconds;
```

Comment:
I don't understand how order of operations works in this expression. For that reason I think the parentheses are useful here even if they make no difference in correctness